card/lib/card/set/format/haml_views.rb

```diff
@@ -84,6 +80,14 @@ def template_location debug_info
           Pathname.new(debug_info[:path])
                   .relative_path_from(Pathname.new(Dir.pwd))
         end
+
+        def with_template_path path
+          old_path = @template_path
+          @template_path = path
+          result = yield
+          @template_path = old_path
```

Comment:
use `ensure`

Reply:
It saves a line and a variable, is more reliable and looks better.

Reply:
Oh, I agree. I was sure I already did this.  Must not have pushed.

Reply:
wait, I did fix this. this is an outdated diff.